bigchaindb/core.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, host=None, port=None, dbname=None,
         self.host = host or bigchaindb.config['database']['host']
         self.port = port or bigchaindb.config['database']['port']
         self.dbname = dbname or bigchaindb.config['database']['name']
+        self.backend = backend or get_backend()
```

Comment:
> I don't think we need self.connection anymore

`self.connection` is still used in some tests. Before updating the tests I want to understand better how to abstract some other database calls. I'll change this part in my next PR, but before I'll put all the details about the refactoring in an issue.